packages/editor/src/client/base.tsx

```diff
@@ -158,7 +170,7 @@ export function Base({children}: BaseProps) {
                   {t('navbar.authors')}
                 </Nav.Item>
 
-                <Nav.Menu eventKey={'1'} title={t('navbar.usersAndMembers')} icon={<PeoplesIcon />}>
+                <Nav.Menu eventKey={'2'} title={t('navbar.usersAndMembers')} icon={<PeoplesIcon />}>
```

Comment:
Out of interest, why the change? I also have no clue what eventKey does, as nothing changes when I press 1 or 2 :D

Reply:
Actually, I don't know, why that changed.